### DISCUSSION TITLE
Attenuation output do not resonate with SED plot

### DISCUSSION THREAD

**Discussion by rocina01** [mboquien/cigale#62] [Q&A]
Hello!
I have encountered an issue that I hopefully you can help me solve.
I've been fitting several objects to derive various parameters and I am particularly interested in the SFR, as I want to compare these results with ones obtained in the past using a different method. The derived data is quite discrepant with what I had previously and looking for a justification for this I have focused on the attenuation and, in doing so, I have detected what seem to me to be discordances with the parameters obtained as output for the attenuation and the SED plot.
For one particular object I get a very low attenuation (att_BC=0; att_ISM=2.6) and yet in the SED there is a significant difference between the 'stellar attenuated' and 'stellar unattenuated' plots.
This actually happens to other objects but this is the most significant case. Hence I was wondering how the attenuation is computed and used to plot the SED as for me it does not seem to match such low attenuation with the model fit.

I attach the 'results' file for this object as well as the best model fit plot.
[10927_best_model.pdf](https://github.com/mboquien/cigale/files/12368095/10927_best_model.pdf)
[results.txt](https://github.com/mboquien/cigale/files/12368115/results.txt)

Thank you very much in advance!
Rocío

**Comment by mboquien** [comment]
Hello Rocío. I would need to see the detail of your `pcigale.ini` file. However, with the BC and ISM attenuations you obtaind, it means that there is no differential reddening, so you should have Av~2.6 mag for all the stellar components. Eyeballing your best fit, it seems to be around that. Essentially with this module, the BC attenuation is applied to the young stellar populations and the ISM component is applied to all the stellar populations (including the young ones).

**Reply by rocina01** [reply]
Ok, it makes sense! Thank you very much Dr Boquien, here is the pcigale.ini anyway.
[pcigale.ini.txt](https://github.com/mboquien/cigale/files/12378081/pcigale.ini.txt)

Thank you again!
Rocío
